smithy-model/src/test/resources/software/amazon/smithy/model/loader/valid/service-with-errors.smithy

```diff
@@ -3,15 +3,17 @@ $version: "1.0"
 namespace ns.foo
 
 service EmptyService {
-    version: "2020-02-18",
-    errors: [
-        Common1,
-        Common2,
-    ],
+    version: "2020-02-18"
+    errors: [Common1, Common2]
```

Comment:
Oh interesting. Is there a threshold for when this uses multiple lines? Service can have lots of errors.